ztools/x509/json.go

```diff
@@ -260,6 +260,21 @@ func (c *Certificate) MarshalJSON() ([]byte, error) {
 			jc.Names = append(jc.Names, name)
 		}
 	}
+	
+	for _, name := range c.URIs {
```

Comment:
Is the `*` in the FQDN or the path? I don't think they're valid in the FQDN, but probably are in the path?